Assets/MRTK/SDK/Features/Utilities/ConstraintExecOrderComparer.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// Defines a comparer to sort TransformConstraints by their
+    /// requested execution order, or any other priority
+    /// mechanism that a subclass utilizes.
+    /// </summary>
+    public class ConstraintExecOrderComparer : IComparer<TransformConstraint>
```

Comment:
Could this be `internal` too, like `ConstraintUtils`?

Reply:
If we make it `internal`, the custom inspector GUI will not be able to access the comparer. I made this futureproof, for if/when we want to improve the custom inspector to also sort/manage the ordering of the constraints.

Reply:
Have you tried InternalsVisibleTo? https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=net-5.0

Reply:
@CDiaz-MS I didn't know we were using this in editor code. I figured it was a tests-only thing.

Reply:
@CDiaz-MS fixed.